### ISSUE TITLE
self.mixer = mixer_cls(dim) TypeError: Mamba.__init__() got an unexpected keyword argument 'bimamba_type'

### ISSUE THREAD

**Issue by Kaolin-Yang** [hustvl/Vim#69]
I encountered another problem when i running the bash train code：
self.mixer = mixer_cls(dim)
TypeError: Mamba.__init__() got an unexpected keyword argument 'bimamba_type'
Have you ever encountered this problem? Do you know how to solve it?

**Comment by ZXXaaaa** [comment]
Hello, I have encountered the same problem, have you solved it?

**Comment by ZXXaaaa** [comment]
Hello, I have encountered the same problem, have you solved it?

**Comment by xianfeizhu** [comment]
> Hello, I have encountered the same problem, have you solved it?

No, I am still trying to resolve it. Can you provide an email address? I am also researching Vision Mamba, so we could exchange ideas on that.

**Comment by xianfeizhu** [comment]
> I think you can see this issue. #67 I don't know whether you install mamba-ssm via `pip install mamba-ssm` . If via this way, you should copy the mamba-ssm dir in Vim to your virtual environment site-packages dir that replace the official mamba-ssm dir. Because the Vim's author modified the mamba-ssm in their code.

This is useful, I have resolved it. Thank you very much.
